msteams-platform/graph-api/activity-feed/test-activity-feed-json-file.md

```diff
@@ -0,0 +1,706 @@
+---
+title: Testing activity feed notifications in Teams
+description: JSON  test file resource-specific consent in Teams using Postman
+localization_priority:  Normal
+author: laujan
+ms.author: lajanuar
+ms.topic: How-to
+keywords: teams authorization OAuth SSO AAD rsc Postman Graph json
+---
+
+# Test RSC Postman collection JSON
```

Comment:
I don't think we should be calling this 'Test RSC' - probably 'Test Activity Feed API Postman collection?